http-api/src/main/java/net/runelite/http/api/item/ItemStats.java

```diff
@@ -81,5 +91,49 @@ public ItemStats subtract(ItemStats other)
 
 		return new ItemStats(quest, equipable, newWeight, 0, newEquipment);
 	}
+
+	public ItemStats add(ItemStats other)
+	{
+		if (other == null)
+		{
+			return this;
+		}
+
+		final double newWeight = weight + other.weight;
+		final ItemEquipmentStats newEquipment;
+
+
+		if (other.equipment != null)
+		{
+			final ItemEquipmentStats equipment = this.equipment != null
+					? this.equipment
+					: NOTHING.getEquipment();
```

Comment:
These lines along with the ItemEquipmentStatsBuilder continuation lines below are indented one level too far.
```suggestion
				? this.equipment
				: NOTHING.getEquipment();
```